Toxygates/src/t/server/viewer/network/ManagedNetwork.scala

```diff
@@ -42,26 +42,27 @@ class ManagedNetwork(mainParams: LoadParams,
                      val sideMatrix: ManagedMatrix,
                      var targets: TargetTable,
                      platforms: PlatformRegistry,
-                     var currentPageSize: Int,
                      sideIsMRNA: Boolean) extends ManagedMatrix(mainParams) {
 
-  protected var currentPageRows: Option[(Int, Int)] = None
+  //Offset and length of the current page in the main matrix, if any
+  var currentPageStart: Int = 0
+  var currentPageLength: Int = current.rows
```

Comment:
The notion of the "current page" in the main matrix is kept for legacy reasons, since it is needed for the dual table view in the GWT client to work. For that case, a ManagedNetwork object is kept in the user's server session in a stateful way. However, from the scalatra API, we only ever load the network once and return it to the client, not tracking user state on the server, and in principle don't need the notion of pages, so instead we load everything as a single "page" that goes from 0 to current.rows (all available rows in the main matrix)